.eslintrc

```diff
@@ -5,7 +5,7 @@
   },
   "extends": "eslint:recommended",
   "parserOptions": {
-    "ecmaVersion": 8
+    "ecmaVersion": 9
```

Comment:
@garethbowen Hi, can I do this to support spread operator in the lint?

Reply:
We support Node 8 which does support the spread operator but it's really hard to prove if it supports _everything_ in ecma 9 so this change risks our linter not catching regressions. I think stick with version 8 for now - we will bump our support matrix to Node 12 (or 14 maybe?) soon so we'll be able to increase this then.

Reply:
Fixed!